prometheus/common/alert-rules.d/crunchy-alert-rules-etcd.yml.example

```diff
@@ -10,11 +10,58 @@ groups:
 
 ########## ETCD EXPORTER RULES ##########
 
+#### IMPORTANT NOTE: These alerts only support monitoring a single etcd cluster per promemtheus instance ####
+
+# Ensure there is at least one etcd node reporting metrics
+  - alert: ECTDAbsent
+    expr: absent(etcd_server_id)
+    for: 10s
+    labels:
+      service: etcd
+      severity: critical
+      severity_num: 300
+    annotations:
+      description: 'No etcd metric endpoints found'
+
+# Ensure there are an odd number of etcd nodes providing metrics
```

Comment:
Do we want odd or we want a quorum? what if you had a 3-node and two were dead?  Or if a 5-node etcd cluster had one node die so there were 4?

Reply:
Not sure how checking for quorum would be different in this case. Number has to be odd right? Latter example it goes to 4 so it would alert.

Reply:
In the case of there only being 1 etcdnode, the `ETCDLeader` alert would fire since no leader could be elected

Reply:
I guess my point was etcd will operate just fine with a quorum, though we'd probably want to alert if we expect we don't have a full set of members (though I guess the "missing" checks below probably catch those cases anyway).  I guess this is not an issue in any case.